docs/csharp/programming-guide/generics/generic-interfaces.md

```diff
@@ -1,44 +1,45 @@
+
+:::code language="csharp" source="./snippets/GenericInterfaces.cs" id="GenericLists":::
+
+Multiple interfaces can be specified as constraints on a single type, as follows:
+
+:::code language="csharp" source="./snippets/GenericInterfaces.cs" id="GenericStack":::
+
+An interface can define more than one type parameter, as follows:
+
+:::code language="csharp" source="./snippets/GenericInterfaces.cs" id="GenericDictionary":::
+
+The rules of inheritance that apply to classes also apply to interfaces:
+
+:::code language="csharp" source="./snippets/GenericInterfaces.cs" id="Months":::
+
+Generic interfaces can inherit from non-generic interfaces if the generic interface is covariant, which means it only uses its type parameter as a return value. In the .NET class library, <xref:System.Collections.Generic.IEnumerable%601> inherits from <xref:System.Collections.IEnumerable> because <xref:System.Collections.Generic.IEnumerable%601> only uses `T` in the return value of <xref:System.Collections.Generic.IEnumerable%601.GetEnumerator%2A> and in the <xref:System.Collections.Generic.IEnumerator%601.Current%2A> property getter.
+
+Concrete classes can implement closed constructed interfaces, as follows:
+
+:::code language="csharp" source="./snippets/GenericInterfaces.cs" id="BaseInterfaces":::
+
+Generic classes can implement generic interfaces or closed constructed interfaces as long as the class parameter list supplies all arguments required by the interface, as follows:
+
+:::code language="csharp" source="./snippets/GenericInterfaces.cs" id="InterfaceInheritance":::
+
+The rules that control method overloading are the same for methods within generic classes, generic structs, or generic interfaces. For more information, see [Generic Methods](./generic-methods.md).
+
+Beginning with C# 11, interfaces may declare `static abstract` or `static virtual` members. Interfaces that declare either `static abstract` or `static virtual` members are almost always generic interfaces. The compiler must resolve calls to `static virtual` and `static abstract` methods at compile time. `static virtual` and `static abstract` methods declared in interfaces don't have a runtime dispatch mechanism analogous to `virtual` or `abstract` methods declared in classes. Instead, the compiler uses type information available at compile time. These members are typically declared in generic interfaces. Furthermore, most interfaces that declare `static virtual` or `static abstract` methods declare that one of the type parameters must [implement the declared interface](../../programming-guide/generics/constraints-on-type-parameters.md#type-arguments-implement-declared-interface). The compiler then uses the supplied type arguments to resolve the type of the declared member.
```

Comment:
This section seems a bit repeated from content that was previously reviewed as part of this PR. If it is exactly the same, I'd recommend creating/using an include file so that you'd only have to make changes to the include, rather than both places later - if needed.

Reply:
I thought about doing that. For now, I want to leave the duplication. Both articles have high page views. In time, I'll be merging the programming guide and language reference sections. That will move this around.